fs_storage/models/fs_storage.py

```diff
@@ -130,6 +130,17 @@ def __init__(self, env, ids=(), prefetch_ids=()):
         help="Relative path to the directory to store the file"
     )
 
+    store_connection = fields.Boolean(
+        default=True,
+        help="If marked, the connection to the filesystem is stored in memory.",
+    )
+    # https://filesystem-spec.readthedocs.io/en/latest/api.html#fsspec.spec.AbstractFileSystem.clear_instance_cache  # noqa: B950
+    invalidate_cache_on_error = fields.Boolean(
```

Comment:
Do we really need a flag for this? IMO we can assume that this is the safest thing to do.